multichat/src/main/java/xyz/olivermartin/multichat/local/common/listeners/LocalLoginLogoutListener.java

```diff
@@ -5,19 +5,24 @@
 
 import xyz.olivermartin.multichat.local.common.MultiChatLocal;
 import xyz.olivermartin.multichat.local.common.MultiChatLocalPlayer;
+import xyz.olivermartin.multichat.local.common.config.LocalConfig;
 
 public abstract class LocalLoginLogoutListener {
 
+	protected String defaultChannel = "global";

```

Comment:
Wouldn't the default channel being a variable make it non-reloadable and require a server restart instead? You can just grab the value from the config every time instead of using this.defaultChannel, since the config is cached anyway and won't affect performance.

Reply:
Ah right, yeah I was worried about this as I didn't know how it worked under the hood so I took the safer option

Reply:
Made these changes now- Wanna have another look?